test/e2e/e2e_gitops_example.go

```diff
@@ -0,0 +1,84 @@
+package test
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/ssh"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	teststructure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestGitopsExample(t *testing.T) {
```

Comment:
So the go testing framework has a somewhat unique rule about what test files can be named. If you currently run the command `go test ./... -run TestGitopsExample -v -timeout 1200s` you will get a response that at a quick glance says passing but if you look a little closer, you'll notice that it didn't actually run the new test you made.

```
(base) ➜  e2e git:(gitops-e2e-test) ✗ go test ./... -run TestGitopsExample -v -timeout 1200s
testing: warning: no tests to run
PASS
ok  	github.com/defenseunicorns/zarf/test/e2e	0.020s [no tests to run]
```


Files that have tests you want to run need to end in `{FILENAME}_test.go` for it to be picked up by the testing framework.